test/unit/calculators/state_pension_age_calculator_test.rb

```diff
@@ -76,6 +148,32 @@ class StatePensionAgeCalculatorTest < ActiveSupport::TestCase
       end
     end
 
+    context '#pension_on_feb_29??' do
+      context 'pension due on 29th of Feb' do
+        setup do
+          @calculator = StatePensionAgeCalculator.new(
+            gender: "male", dob: Date.parse("29 Feb 1980"))
+          @calculator.stubs(:state_pension_date).returns(Date.parse('29 Feb 2048'))
+        end
+
+        should "be true for a date that is the 29th of Feb" do
+          assert_equal true, @calculator.pension_on_feb_29?
```

Comment:
We should use `assert` here which fails unless the supplied argument is a `true` value.

```ruby
assert @calculator.pension_on_feb_29?
```

See http://ruby-doc.org/stdlib-2.0.0/libdoc/minitest/rdoc/MiniTest/Assertions.html#method-i-assert

Reply:
Noted